workbench_reviewer.permissions.yml

```diff
@@ -0,0 +1,12 @@
+assign workbench reviewer:
+  title: 'Assign Workbench Reviewer'
+  description: 'Users with this permissions may view and access the <em>Workbench Reviewer</em> form to assign content to other users.'
+  restrict access: FALSE
+be assigned workbench reviewer:
+  title: 'Be assigned Workbench Reviewer'
+  description: 'Users with this permissions may be assigned content to review via <em>Workbench Reviewer</em>.'
+  restrict access: FALSE
+administer workbench reviewer:
```

Comment:
Unused at the moment, but will be used later when there are more Workbench Reviewer options / configuration.